src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -27,12 +34,21 @@
 
 public class HologramProjector extends SlimefunItem {
 
-    private static final String OFFSET_PARAMETER = "offset";
+    private final String DEFAULT_OFFSET = "0.5";
```

Comment:
This should rather be static, since it is a constant anyway.

```suggestion
    private static final String DEFAULT_OFFSET = "0.5";
```